### PR TITLE
[NativeAOT-LLVM] A couple of missing symbol fixes

### PR DESCRIPTION
**Pull request by SingleAccretion**
Mainly EventSource-related fixes.

Contributes to #2454.

### CODE REVIEW

File: src/coreclr/nativeaot/System.Private.CoreLib/src/System.Private.CoreLib.csproj [hunk 1]
```diff
@@ -190,7 +190,7 @@
     <Compile Include="System\Diagnostics\StackTrace.NativeAot.Browser.cs" Condition="'$(TargetOS)' == 'browser'" />
     <Compile Include="System\Diagnostics\StackTrace.NativeAot.Wasi.cs" Condition="'$(TargetOS)' == 'wasi'" />
     <Compile Include="System\Diagnostics\Eventing\EventPipe.NativeAot.cs" />
-    <Compile Include="System\Diagnostics\Eventing\NativeRuntimeEventSource.Threading.NativeSinks.NativeAot.cs" />
+    <Compile Include="System\Diagnostics\Eventing\NativeRuntimeEventSource.Threading.NativeSinks.NativeAot.cs" Condition="'$(FeaturePerfTracing)' == 'true'" />
```
Comment (SingleAccretion) [inline src/coreclr/nativeaot/System.Private.CoreLib/src/System.Private.CoreLib.csproj:193]: https://github.com/dotnet/runtime/pull/96811

File: src/coreclr/nativeaot/Runtime/wasm/PalRedhawkWasm.cpp [hunk 2]
```diff
@@ -204,13 +204,8 @@ REDHAWK_PALEXPORT _Ret_maybenull_ _Post_writable_byte_size_(size) void* REDHAWK_
...
     }
 
-    // The Unix version of this function allocates memory with 64K alignment, even as nothing needs such large
-    // alignments as of the writing of this comment. We do have to return something at least page-aligned.
-    const unsigned Alignment = 64 * 1024;
-    ASSERT(OS_PAGE_SIZE <= Alignment);
-
     void* pRetVal;
-    if (posix_memalign(&pRetVal, Alignment, size) != 0)
+    if (posix_memalign(&pRetVal, OS_PAGE_SIZE, size) != 0)
```
Comment (SingleAccretion) [inline src/coreclr/nativeaot/Runtime/wasm/PalRedhawkWasm.cpp:208]: This alignment compatibility quirk was removed upstream.

**Comment by SingleAccretion** [comment]
@dotnet/nativeaot-llvm

**Review by yowl — APPROVED** [review]
Thank!

**Pull request merged**